backends/lean/Base/Progress/Progress.lean

```diff
@@ -392,9 +394,12 @@ elab tk:"progress?" args:progressArgs : tactic => do
   let stats ← evalProgress args
   let mut stxArgs := args.raw
   if stxArgs[1].isNone then
-    let withArg := mkNullNode #[mkAtom "with", mkIdent stats.usedTheorem]
+    let withArg := mkNullNode #[mkAtom "with", stats.usedTheorem]
     stxArgs := stxArgs.setArg 1 withArg
   let tac := mkNode `Progress.progress #[mkAtom "progress", stxArgs]
+  -- if stats.usedTheorem.isInaccesisble then
+  --  logInfo "{...} is inaccessible, if you want to actually use the suggestion, please use `rename_i ...`"
```

Comment:
Please remove all commented code (note however that printing such a comment would be useful, if it is possible).